Care-CureHospital/Backend/Model/DoctorMenager/Report.cs

```diff
@@ -0,0 +1,62 @@
+﻿using HealthClinic.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Backend.Model.DoctorMenager
+{
+    public class Report : IIdentifiable<int>
+    {
+        public int Id { get; set; }
+        
+        public String MedicationName { get; set; }
+
+        public int Quantity { get; set; }
+
+        public DateTime Date { get; set;  }
+
+        public Report(int id, string name, int quantity)
+        {
+            this.Id = id;
+
```

Comment:
You should not put keyword 'this' in constructor if name of constructor attribute is different than attribute of class itself.